src/plugins/console/public/lib/autocomplete/components/username_autocomplete_component.js

```diff
@@ -17,14 +17,14 @@
  * under the License.
  */
 import _ from 'lodash';
-import mappings from '../../mappings/mappings';
+import { getIndices } from '../../mappings/mappings';
```

Comment:
Is tree shaking not applicable to the old import style?

Reply:
It will include the entire object. In the case of mappings, it was an object containing all the functions which wouldn't be able to be trimmed.

We also disallow default exports.